scripts/package_building/deps-lis/rpm/perf/lis-perf.spec

```diff
@@ -33,3 +33,7 @@ mv %{buildroot}%{_usr}/etc %{buildroot}
 %{_bindir}/trace
 %{_usr}/share/*
 %{_usr}/lib64/libperf-jvmti.so
+
+# Ver:4.17.0
```

Comment:
the 4.17.0 version is incomplete, as the change landed in between rc's and not when 4.17.0 came,
can you check exactly during which rc this changed and add the rc version too?